.github/workflows/terraform.yaml

```diff
@@ -2,6 +2,11 @@ name: terraform-ci
 on:
   workflow_call:
     inputs:
+      only_fmt_validate:
+        description: Run only format and validate
+        type: boolean
+        default: false
+        required: false
```

Comment:
We can flip this over to 

- `enable_format`: default true
- `enable_lint`: default true
- `enable_security_checks`: default true